helmfile.d/snippets/templates.gotmpl

```diff
@@ -2,23 +2,27 @@ templates:
   default: &default
     missingFileHandler: Debug
     chart: ../charts/{{ .Release.Name }}
+    disableValidationOnInstall: true
```

Comment:
Wrong place. Please put it in the defaults ;)

Reply:
that works too, but I changed it now.
For setting it in the defaults file, the property is called `disableValidation`.
It is fine if we want to have them preset as true for every release, then override it if needed on release level.

Reply:
> Ok, image with version `1.4.11` is built and pushed but there is another error regarding `ajv` tool.
> 
> ```
> error: parameter --extend-refs is unknown
> usage:
>     validate:  ajv [validate] -s schema[.json] -d data[.json]
> ```
> 
> @j-zimnowoda can you please take a look why it is failing now?

On it

Reply:
The ajv 7.0.0 has removed `--extend-refs` option. https://openbase.com/js/ajv/versions

> extendRefs: $ref siblings are validated by default (consistent with draft 2019-09), ignoreKeywordsWithRef (deprecated) can be used instead to ignore $ref siblings.

Reply:
ok, then I am changing it to install fixed version 3.3.0, this was used before the breaking change.